### PR TITLE
Select sum23 bug fixes

### PR DESCRIPTION
**Pull request by amgrubb**
Replaces PR #725 

@kkung9 @mollyd13  Please reconfirm via testing that bugs #708, #667, #687, #686, and #663 are fixed successfully. 

@YesugenB Please test and check next state changes, and EVO slider appearing disappearing correctly.

### CODE REVIEW

**Review by YesugenB — COMMENTED** [review]
EVO slider does not work properly in Analysis Mode when a path has been simulated. 

1. When EVO is turned off and you go into Analysis mode to simulate a path and turn on EVO into % or Time mode, the slider does not disappear. 
2. When EVO is turned on and you go into Analysis mode to simulate a path, the default is EVO % mode and the slider does not appear, which is good. However, when you go to EVO State Mode, the slider does not appear when it should. But when you turn EVO off, the slider appears. Then it does not disappear when you go back to Time or % mode.

File: leaf-ui/js/next-state.js [hunk 1]
```diff
@@ -197,6 +197,14 @@
         var pagination = document.getElementById("pagination");
         var num_states_lbl = document.getElementById("num_states_lbl");
         var currentPageIn = document.getElementById("currentPage");
+
+        // disable Explore Next States button if last time point
+        var exploreNextStatesBtn = document.getElementById("exploreNextStates");
```
Comment (YesugenB) [inline leaf-ui/js/next-state.js:202]: If the last time point is selected, the Explore Next States button is disabled. However, it is not re-enabled when a different, earlier time point is selected. This remains even when you load a new Explore States window from the same time point.
Screen Shot 2023-10-19 at 1 08 23 PM

**Review by kkung9 — COMMENTED** [review]
Specified issues are addressed. The EVO slider issue mentioned above is fixed in the Presence Conditions branch but not in this branch.

**Comment by mollyd13** [comment]
Issues #708, #667, #687, #686, and #663 seem to be resolved. See Yesugen's comment for EVO slider and next state changes.

**Comment by amgrubb** [comment]
I pulled in the changes from develop, which adds presence conditions. Let's re-test everything and make sure the bugs are fixed.

File: leaf-ui/js/next-state.js (hunk 1 above)
Comment (mollyd13) [reply leaf-ui/js/next-state.js:202]: @YesugenB We are having trouble recreating this issue. Could you clarify what you mean by "selecting an earlier timepoint." Do you mean closing the next state tab and reopening a new one at an earlier timepoint from the analysis tab? Or is there a way to get to an earlier timepoint from the next state window?

**Review by mollyd13 — APPROVED** [review]
Bugs are still fixed, and slider works as expected.

**Pull request merged**